components/atom/spinner/src/DefaultSpinner.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types'
+import {forwardRef} from 'react'
+
+const DefaultSpinner = forwardRef(({delayed, loader}, forwardedRef) => {
+  return <span ref={forwardedRef}>{!delayed ? loader : <noscript />}</span>
+})
+
+DefaultSpinner.propTypes = {
+  delayed: PropTypes.bool,
```

Comment:
since we're releasing a major... 

```suggestion
  isDelayed: PropTypes.bool,
```

what do you think?

Reply:
boolean props might have the prefix:
✅  `is[whatever]`
✅  `has[whatever]`

❌  `with[whatever]` because in react, with prefix is used for HoC
❌  `use[whatever]` because in react, use prefix is used for hooks

Reply:
Good idea! Thanks!